src/Documentation/grains/index.md

```diff
@@ -5,11 +5,12 @@ title: Developing a Grain
 
 # Setup
 
-Before you write code to implement a grain class, create a new Class Library project targeting .NET 4.6.1 or higher in Visual Studio and add the `Microsoft.Orleans.CodeGenerator.MSBuild` NuGet package to it.
+Before you write code to implement a grain class, create a new Class Library project targeting .NET Standard (preferred) or .NET Framework 4.6.1 or higher (if you cannot use .NET Standard due to dependencies).
+Grain interfaces and grain classes can be defined in the same Class Library project or in two different projects for better separation of interfaces from implementation.
+In either case, the projects need to reference `Microsoft.Orleans.Core.Abstractions` and `Microsoft.Orleans.CodeGenerator.MSBuild` NuGet packages.
+
+For more thorough instructions, see the [Project Setup](../tutorials_and_samples/tutorial_1.md#project-setup) section of [Tutorial One – Orleans Basics](../tutorials_and_samples/tutorial_1.md).
```

Comment:
I changed these to relative links to the md files instead of the HTML files: https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html